README.md

```diff
@@ -24,6 +24,11 @@ Example Configuration, add to settings.yaml
         value: "lorem ipsum #{host.host_attribute_name}"
     :delete
       :StatusID: integer_id
+    :search:
+      AppID: 741
```

Comment:
Is this the same AppID under uri: id?

If so, lets call that uri: AppID and just use that here too.

Reply:
I don't want to specify the search params like this in yaml. Anything you NEED should be under api: 

The only think I want to search on is for a host matching by teamdynamix_asset_id or name/serialnumber in a loop over all hosts. See my other comment